weyert.ReminderHelpers/src/reminders.js

```diff
@@ -0,0 +1,368 @@
+// @flow
+
+// ------------------------------------------------------------------------------------
+// Command to turn tasks into full reminder events
+// @weyert
+// v0.1.0, 7.8.2021
+//
+// ------------------------------------------------------------------------------------
+
+import { getOrMakeConfigurationSection } from '../../nmn.Templates/src/configuration'
+
+import {
+  defaultTodoCharacter,
+  displayTitle,
+  showMessage,
+  dateStringFromCalendarFilename,
+  toISODateString,
+  smartPrependPara,
+} from '../../helperFunctions'
+
+// ------------------------------------------------------------------------------------
+const DEFAULT_REMINDERS_CONFIG = {}
+
+const minimumConfig = {}
+
+function getDateFromString(dateStr: string): Array<int> {
```

Comment:
`Array<number>`
Javascript doesn't have int. Please make sure you have flow running locally.

Reply:
Oh okay, I will do :D